cmd/vela-terraform/plan.go

```diff
@@ -133,11 +133,10 @@ func (p *Plan) Command(dir string) *exec.Cmd {
 	if len(p.Vars) > 0 {
 		var vars string
 		for _, v := range p.Vars {
-			vars += fmt.Sprintf(" %s", v)
+			vars += fmt.Sprintf("-var=%s ", v)
 		}
-
-		// add flag for Vars from provided plan command
-		flags = append(flags, fmt.Sprintf("-var=\"%s\"", strings.TrimPrefix(vars, " ")))
+		// add flag for Vars from provided validate command
```

Comment:
Is this comment change accurate?

Reply:
No - reverted the mistake from my copy/paste.